requirements.in

```diff
@@ -1,11 +1,11 @@
-Django<3.3                        # core server-side framework
+Django==4.2.13                    # core server-side framework
 django-anymail                    # for sending emails using cloud-based mail service providers
 django-csp                        # for setting CSP headers
 django-guardian                   # object-level permissions for database records. Behind a major version due to missing Python 2 support
 django-hijack                     # allows admins to login as other user
 django-notifications-hq           # notification app
 django-cors-headers               # allows CORS requests for client-side development
-django-storages[google]==1.11.1   # alternative GCS storage backend for the django media_root
+django-storages[google]==1.14.3   # alternative GCS storage backend for the django media_root
```

Comment:
in general we do not pin dependencies here to patch versions, although it seems like when we set this up we had to for this one library to get it to compile - if you remove the version for this now can you get a vaid build, or is it still being picky

Reply:
I removed the pinned version of django-storages, ran pip-compile and got no change to requirements.txt from what's in master now (it'll stay 1.11.1). 

We use django-storages for uploading pedigree images to gcs. I tested this codepath via the family_api unit tests and setting the 'GCS_MEDIA_ROOT_BUCKET' to 'seqr-media-storage-public-prod' - the pedigree uploaded to gcs with no issues.